packages/befunge93/1.0.0/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+source ../../python/2.7.18/build.sh
```

Comment:
Can we use python3?

Reply:
i don't think so because they used print without brackets `print "File too large"`